packages/typescriptlang-org/src/copy/en/index.ts

```diff
@@ -38,7 +38,7 @@ export const indexCopy = {
 
   index_dts: "Types On Every Desk",
   index_dts_copy: `
-<p>Most of the worlds JavaScript’s is un-typed, and inference can only go so far. To address this, the TypeScript team helps maintain <dt>Definitely Typed</dt> - a community project to provide types and inline documentation to existing JavaScript.
+<p>Most of the worlds JavaScript is un-typed, and inference can only go so far. To address this, the TypeScript team helps maintain <dt>Definitely Typed</dt> - a community project to provide types and inline documentation to existing JavaScript.
```

Comment:
```suggestion
<p>Most of the world's JavaScript is un-typed, and inference can only go so far. To address this, the TypeScript team helps maintain <dt>Definitely Typed</dt> - a community project to provide types and inline documentation to existing JavaScript.
```

Reply:
@orta, does the site use smart or simple apostrophes throughout?

Reply:
simple